Cargo.toml

```diff
@@ -17,4 +17,5 @@ members = [
     "components/uniset",
     "resources/testdata",
     "utils/fixed_decimal",
+    "tools/release",
```

Comment:
I'm not sure about the organization of another top-level directory.  We have several tools, such as icu4x-gen-testdata and icu4x-cldr-export, and those tools currently live in their associated crate.  Do you suggest we move those tools into the new top-level tools directory?  Or, is there a particular component to where you think we can move the "release" tool?

Reply:
My thinking is that `components` are the things that people use, `utils` are helper APIs, and `tools` are binaries we use to manage ICU4X.

I expect to grow this tool to add more pieces of release management.

Reply:
I guess the premise with which I don't fully agree is that there's such a thing as "release management" relating to code.  After v1, any code that is checked-in to the repo should be stable.  All development work should be done in PRs and non-main branches.  README generation should be an automated step in a CI or something.

Reply:
Basically, releasing a new version of ICU4X should be no harder than `cargo release`.  We should not create tools that for release managers.  If we're tempted to create them, we are doing something wrong, and it should be a CI step instead.

Reply:
so, there's a bit more to release:
* update readmes
* build changelog from diff
* push all crates to repos - https://crates.io/crates/cargo-unleash

Granted, we could separate the first two to CI actions that work continuously and then what's left is `cargo-unleash`.

In which case, maybe the name of the tool should be `tools/extract-readme` ?

Reply:
Does https://crates.io/crates/cargo-release help?

Reply:
yes!

Reply:
Are you ok with me renaming it `tools/extract-readme` ?

Reply:
@sffc ?

Reply:
Yes please.  I like that better.

It would be nice to generalize this tool in such a way that we can pull it out into its own repo, and remove process tools from the main ICU4X repo.